charts/longhorn/102.3.0+up1.5.1/templates/validate-install-crd.yaml

```diff
@@ -0,0 +1,33 @@
+#{{- if gt (len (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" "")) 0 -}}
+# {{- $found := dict -}}
+# {{- set $found "longhorn.io/v1beta1/BackingImageDataSource" false -}}
+# {{- set $found "longhorn.io/v1beta1/BackingImageManager" false -}}
+# {{- set $found "longhorn.io/v1beta1/BackingImage" false -}}
+# {{- set $found "longhorn.io/v1beta1/Backup" false -}}
+# {{- set $found "longhorn.io/v1beta1/BackupTarget" false -}}
+# {{- set $found "longhorn.io/v1beta1/BackupVolume" false -}}
+# {{- set $found "longhorn.io/v1beta1/EngineImage" false -}}
+# {{- set $found "longhorn.io/v1beta1/Engine" false -}}
+# {{- set $found "longhorn.io/v1beta1/InstanceManager" false -}}
+# {{- set $found "longhorn.io/v1beta1/Node" false -}}
+# {{- set $found "longhorn.io/v1beta2/Orphan" false -}}
+# {{- set $found "longhorn.io/v1beta1/RecurringJob" false -}}
+# {{- set $found "longhorn.io/v1beta1/Replica" false -}}
+# {{- set $found "longhorn.io/v1beta1/Setting" false -}}
+# {{- set $found "longhorn.io/v1beta1/ShareManager" false -}}
+# {{- set $found "longhorn.io/v1beta2/Snapshot" false -}}
+# {{- set $found "longhorn.io/v1beta2/SupportBundle" false -}}
+# {{- set $found "longhorn.io/v1beta2/SystemBackup" false -}}
+# {{- set $found "longhorn.io/v1beta2/SystemRestore" false -}}
+# {{- set $found "longhorn.io/v1beta1/Volume" false -}}
```

Comment:
Miss the new CRD `volumeattachments.longhorn.io` https://github.com/longhorn/longhorn/blob/19e8fefd3ace7fb66c3f3521fc471b60a829b155/chart/templates/crds.yaml#L3555